package.json

```diff
@@ -98,9 +99,12 @@
     "babel-loader": "8.1.0",
     "babel-preset-expo": "8.2.3",
     "eslint": "6.7.2",
+    "eslint-config-prettier": "^6.12.0",
```

Comment:
Could you remove all the ^ from package.json? To make sure we all have the same version

Reply:
Done